src/framework/model.js

```diff
@@ -279,11 +279,11 @@ class RaModelBase extends GObject.Object {
             let corruption;
             try {
                 corruption = (await gameState.Get(key)).deep_unpack();
+                const state = corruption.state.deep_unpack();
```

Comment:
This won't work, `state` will be declared locally to the `try` block and won't exist outside it. You'll need to declare it outside of the block and assign it inside the block.

Reply:
Done.